tests/test_run_length.py

```diff
@@ -79,7 +79,12 @@ def test_start_at_0(self):
 
         # n-dim version versus ufunc
         da3d = xr.open_dataset(self.nc_pr).pr[:, 40:50, 50:68] * 0
-        da3d[0:10,] = da3d[0:10,] + 1
+        da3d[0:10,] = (
+            da3d[
+                0:10,
+            ]
+            + 1
+        )
```

Comment:
Yo, que pasa? I'm not sure I understand which pep8 rule the line was breaking? It's not the only one-liner that is now spread out for no apparent reason...